### DISCUSSION TITLE
regularization

### DISCUSSION THREAD

**Discussion by saribolagh** [JaidedAI/EasyOCR#592] [General]
While training I see very low training loss and very high validation loss. What are the options to regularize the model training?
